src/views/CardSceneView.js

```diff
@@ -19,38 +19,36 @@ type Props = {
 };
 
 export default class SceneView extends React.PureComponent<Props> {
-
   constructor(props: DFSCardNavigatorProps) {
     super(props);
     (this: any)._trackState = this._trackState.bind(this);
   }
 
   _trackState(route, data) {
     if (!route.trackingPageName) {
-      console.warn(`Attribute \`trackingPageName\` not defined in routes for ${route.routeName}`);
+      console.warn(
```

Comment:
@caseycookie14 did we get the right branch?  Changes here aren't intentional are they?

Looks like we're losing some stuff.

Reply:
@sdg9 yeah i think so, this change is in the first commit you made on your [flipAnimation2 branch](https://github.com/CaribouBot/react-navigation/pull/4/commits/f937af5007744d5e388f872520d816b1e607730c)